tools/api-builder/angular.io-package/processors/addJadeDataDocsProcessor.js

```diff
@@ -130,10 +135,10 @@ module.exports = function addJadeDataDocsProcessor() {
               title: exportDoc.name,
               docType: exportDoc.docType,
               exportDoc: exportDoc,
-              stability: stability,
-              howToUse: howToUse,
-              whatItDoes: whatItDoes,
-              security: security
+              stability: escapeQuotes(stability),
+              howToUse: escapeQuotes(howToUse),
+              whatItDoes: escapeQuotes(whatItDoes),
+              security: escapeQuotes(security)
```

Comment:
I don't like doing the replacement here in the code.
We don't necessarily know at this point what the output rendering file format would be...
It is better to do the escaping in the template at `/angular.io/tools/api-builder/angular.io-package/templates/api-list-data.template.html`